### PR TITLE
Error warning messages

### PR DESCRIPTION
**Pull request by chrispymm**
* Adds in new warning message style for blocking errors on the publishing page.
* Fixes error where @autocomplete_items was `nil` on the `publish_controller#create` action

image

### CODE REVIEW

File: app/controllers/publish_controller.rb [hunk 1]
```diff
@@ -1,11 +1,10 @@
 class PublishController < FormController
-  before_action :assign_form_objects
+  before_action :assign_form_objects, :assign_autocomplete_warning
```
Comment (brenetic) [inline app/controllers/publish_controller.rb:2]: is this because it is also required for the `create`?

Comment (brenetic) [reply app/controllers/publish_controller.rb:2]: just a thought, but to save making the API call until only when it's necessary we could also do this just before we render `:index` on line 20.

**Review by brenetic — APPROVED** [review]
🚀 

one comment, happy for it to stay as is if you prefer

File: app/controllers/publish_controller.rb (hunk 1 above)
Comment (njseeto) [reply app/controllers/publish_controller.rb:2]: oh yeah this is a good suggestion. I don't have a strong preference out of the two options, happy with either

**Review by njseeto — APPROVED** [review]

**Pull request merged**